vmdb/app/models/mixins/password_mixin.rb

```diff
@@ -13,6 +18,9 @@ def encrypt_column(column)
       #   password_encrypted  : Get the password in cryptext
       #   password_encrypted= : Set the password in cryptext
 
+      encrypted_columns << column.to_s
+      encrypted_columns << "#{column}_encrypted"
```

Comment:
> not sure that column_encrypted is an encrypted_columns. Thought only column.to_s would be in there

Interesting point, and maybe that means the method name is bad.  I too would expect that a method named `encrypted_columns` would not contain `column_encrypted`.  The use case for this method is really being able to filter specific values from the REST API, and we definitely want to filter the column_encrypted.

This is tricky...maybe it just needs a better name, or perhaps you need two separate methods.